### ISSUE TITLE
MS1 precursor ion/s with 34S isotope pattern

### ISSUE THREAD

**Issue by Mattesimone** [mwang87/MassQueryLanguage#213]
Hi,

I run massql tool (Commandline) to search for sulfur-containing signals with the following query: 
QUERY scaninfo(MS1DATA) WHERE MS1MZ=X AND MS1MZ=X+1.996:TOLERANCEPPM=2
I'm attaching the unexpected output in which the difference between  mz_lower and  mz_upper mz is always  20.

Then I execute the same MassQL query against my data at GNPS but the job failed: ID=be0a38f7bdd2472da1f5cd5b4d58bdfc
![Screenshot 2022-03-18 at 14 37 41](https://user-images.githubusercontent.com/56867271/159012895-5b43d5fc-fbbf-464b-9cc5-1edea9edd9b1.png)

Thanks in advance for help

Matteo

**Comment by mwang87** [comment]
Yes, your job just took too long on GNPS so it was killed. The mz_upper and mz_lower, are by definition above and below the "comment" which is the value of the X you found. Its mostly been used to make rendering much easier, though likely it should probably belong in an accompanying script rather than main MassQL. 

To make it run faster, I'd recommend setting minimum peak intensity thresholds for your X peak. 

```
QUERY scaninfo(MS1DATA) WHERE MS1MZ=X:INTENSITYPERCENT=5 AND 
MS1MZ=X+1.996:TOLERANCEPPM=2
```

**Comment by mwang87** [comment]
Awesome! Good luck with it and feel free to ask questions and contribute.
